python/src/aac/plugins/generate/generate_impl.py

```diff
@@ -306,7 +318,17 @@ def generate(
 def clean(
     aac_file: str, code_output: str, test_output: str, doc_output: str, no_prompt: bool
 ) -> ExecutionResult:
-    """Clean up generated code, tests, and docs."""
+    """
```

Comment:
Looks like the description got dropped.
```suggestion
    """
    Clean up generated code, tests, and docs.
```